test/itg/1.7/globals.webdriverio.js

```diff
@@ -90,16 +90,16 @@ module.exports = {
         //order_reference: '#content > div.row > div > div:nth-child(5) > div.col-lg-7 > div:nth-child(1) > div.panel-heading > span:nth-child(2)',
         order_reference: '((//div[@class="panel-heading"])[1]/span)[1]',
         modules_menu: '#subtab-AdminParentModulesSf',
-        modules_search: '.pstaggerAddTagInput.module-tags-input',
-        modules_search_button: '.input-group-addon.module-search-icon',
+        modules_search: '#main-div > div.content-div > div > div > div.row > div > div.module-top-menu > div > div.col-md-8 > div > div > div.pstaggerAddTagWrapper > input',
```

Comment:
The selector seems quite long. Did you try with `modules_search: 'div.pstaggerAddTagWrapper > input',`?